src/css/objects/_main-box.scss

```diff
@@ -0,0 +1,11 @@
+.main-box {
```

Comment:
The name `main-box` is so subjective. Please look at naming conventions used by popular css library, bootstrap, foundation, etc. Rename it accordingly so that other contributors can understand its usage.